build.sbt

```diff
@@ -123,6 +150,27 @@ lazy val checkReleaseType: ReleaseStep = ReleaseStep({ st: State =>
   st
 })
 
+lazy val releaseCIProcessSteps: Seq[ReleaseStep] = {
+  val commonSteps = Seq(
+    // seems like we don't need this next step for the CI release process, we already know we're doing a snapshot release
```

Comment:
Some of these comments will become superfluous, but I've retained them for now as I think it'll help remind us (and work out what's needed) when it comes to defining the release process (non-snapshot releases).